seatunnel-connectors-v2/connector-kudu/src/main/java/org/apache/seatunnel/connectors/seatunnel/kudu/source/KuduSourceSplitEnumerator.java

```diff
@@ -39,11 +40,19 @@ public class KuduSourceSplitEnumerator implements SourceSplitEnumerator<KuduSour
     public KuduSourceSplitEnumerator(SourceSplitEnumerator.Context<KuduSourceSplit> enumeratorContext, PartitionParameter partitionParameter) {
         this.enumeratorContext = enumeratorContext;
         this.partitionParameter = partitionParameter;
+        this.parallelism = enumeratorContext.currentParallelism();
```

Comment:
`EnumeratorContext.currentParallelism()` is Runtime Method, Use it in `SplitEnumerator#run()`

Reply:
Please do not save the value returned by `EnumeratorContext.currentParallelism()`